server/middlewares/articles/findArticle.js

```diff
@@ -6,21 +6,20 @@ import { serverError, httpResponse } from "@helpers/http";
 
 const { Articles } = models;
 export const findArticle = async (req, res, next) => {
+  const slug = req.params.slug || req.body.slug;
```

Comment:
please it is from params not from the body

Reply:
The rate Article makes uses of POST request with additional body "score" It is permissible and advisable to include post request data in the request body. This route covers for both case if the data is included as url parm or post body

Reply:
we did not agree as a team to do that